src/clj/commiteth/bounties.clj

```diff
@@ -127,6 +151,7 @@
     (if-let [merged-or-paid? (or (:winner_login bounty)
                                  (:payout_hash bounty))]
       (cond
+        (:payout_receipt bounty)        :paid-with-receipt
```

Comment:
Nice. I see you are using this elsewhere as the flag for when to generate the paid comment. This is great as I always felt that previously we were using `payout_hash` as a basis for `paid` in the UI and that was overly optimistic until we had the receipt in hand. 

FYI I had changed the definition of paid in my other PR to use receipt instead of hash but it should straightforward to use this new definition. 👍